### ISSUE TITLE
Custom zoom values

### ISSUE THREAD

**Issue by NathanVeeva** [jkgenser/react-pdf-headless#22]
I am looking at porting from `react-pdf-viewer` due to the owner no longer maintaining that library. 

I believe you've done a great job with this lib. Though I was wondering, is it possible to pass in a zoom level. IE: 0.5, 1, 2, etc.

**Comment by jkgenser** [comment]
@NathanVeeva : Yes it is possible to change zoom level. It default renders to fit page width based on the parent container. Right now you need to instantiate a readerAPI handle and pass it in for the component to initialize once the document is loaded.

Once the readerAPI is initialized, you can then imperatively call zoom in or zoom out. There's no current way to set a default zoom scale to the component itself however. I will add that to the list of upcoming features!

https://github.com/jkgenser/react-pdf-headless/blob/main/demo/App.tsx#L168

I've added the feature request to the roadmap: https://github.com/jkgenser/react-pdf-headless/issues/23
